libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -447,6 +446,7 @@ - (void) cleanResyncGhosts:(NSNumber*)syncId completionStatusBlock:(SFSyncSyncMa
         [self log:SFLogLevelError format:@"Cannot run cleanResyncGhosts:%@:wrong type:%@", syncId, [SFSyncState syncTypeToString:sync.type]];
         return;
     }
+    [self logAnalyticsEventWithSyncState:sync name:@"cleanResyncGhosts" startTime:[[NSDate date] timeIntervalSince1970]];
```

Comment:
Just as a measure of housekeeping: since all of these specify the same value for the startTime parameter, should that value simply be rolled into the implementation of the method?